embc-app/Controllers/VolunteersController.cs

```diff
@@ -50,7 +50,7 @@ public async Task<IActionResult> Get(string id)
         [HttpPost]
         public async Task<IActionResult> Create([FromBody] Volunteer item)
         {
-            var existing = dataInterface.GetVolunteerByExternalId(item.Externaluseridentifier);
```

Comment:
no idea why this is included in this commit..

Reply:
@ytqsl that commit above is to deal with duplicate bceid usernames when creating ESS users

Reply:
I know, but it is not part of this story, git included it in the diff

Reply:
yeah, weird. Did you rebase?

Reply:
It's not the right code either?

Reply:
I'll rebase again and update